managed/ui/src/components/config/Security/KeyManagementConfiguration.js

```diff
@@ -275,18 +276,23 @@ class KeyManagementConfiguration extends Component {
 
   render() {
     const { configList } = this.props;
-    const { listView, enabledIAMProfile } = this.state;
+    const { listView, enabledIAMProfile} = this.state;
+
     if (getPromiseState(configList).isInit() || getPromiseState(configList).isLoading()) {
       return <YBLoadingCircleIcon />;
     }
     if (listView) {
       return (
-        <ListKeyManagementConfigurations
-          configs={configList}
-          onCreate={this.openCreateConfigForm}
-          onDelete={this.deleteAuthConfig}
-        />
+        <React.Fragment>
```

Comment:
You can use `<> </>` instead of `React.Fragment`

Reply:
Noted, I am leaving it as is for now, let's hear from others

Reply:
Let's use `<></>` syntax pls

Reply:
fragment seems not needed here anymore

Reply:
Done